src/app/components/applications/SSV/RegisterValidatorHome/components/SelectOperators/components/FirstSquare/FirstSquare.tsx

```diff
@@ -95,30 +91,34 @@ const FirstSquare = ({ editPage, clusterSize, setClusterSize, clusterBox }: {
     setOperatorsPagination(response.pagination);
   };
 
-  const selectOperator = (e: any, operator: IOperator) => {
+  const selectOperatorHandling = (e: any, operator: IOperator) => {
     // @ts-ignore
```

Comment:
remove ts-ignore

Reply:
it's make a error, I will refactor all this component soon